docs/scripts/reference.py

```diff
@@ -524,7 +531,7 @@ def gen_reference_commands():
             )
 
 
-def gen_reference_templatetags():
+def gen_reference_template_tags():
```

Comment:
Unrelated to the bug, but renamed the functions / file names in the generated docs so we use full names like "template variables" instead of "template vars"